soc/xtensa/intel_adsp/cavs_v25/linker.ld

```diff
@@ -25,10 +25,27 @@ OUTPUT_ARCH(xtensa)
 PROVIDE(__memctl_default = 0x00000000);
 PROVIDE(_MemErrorHandler = 0x00000000);
 
-#define RAMABLE_REGION ram :ram_phdr
-#define ROMABLE_REGION ram :ram_phdr
 #define LP_SRAM_REGION lpram
 
+/* DSP RAM regions (all of them) are mapped twice on the DSP: once in
```

Comment:
Where can I look, in the code, to see the mapped done twice?
Where is the mapping done?

Reply:
@iuliana-prodan This is feature of the hardware, it provides aliased view to RAM at a different address.

Reply:
Is this due to how Xtensa DSP core was integrated on Intel's platforms?

Reply:
@iuliana-prodan Not sure, just copying what we do in SOF firmware with its own RTOS.